README.md

```diff
@@ -48,7 +48,7 @@ buildscript {
 }
 
 repositories {
-    maven { url  "http://palantir.bintray.com/releases" }
+    mavenCentral()
```

Comment:
I followed the approach taken in #1728, but I am not myself a Gradle user, so perhaps this is the default configuration, in which case this section could be dropped :shrug:.